src/plugins/data/public/ui/filter_bar/filter_editor/index.tsx

```diff
@@ -108,7 +108,7 @@ class FilterEditorUI extends Component<Props, State> {
   public render() {
     return (
       <div>
-        <EuiPopoverTitle>
+        <EuiPopoverTitle paddingSize="s">
```

Comment:
callout: placing padding on popovertitle because there isn't a popover to place this on (css change handles the body)